Update workers.conf.j2 (#6163)
edx/configuration: CHANGELOG.md

All notable changes to this project will be documented in this file.
Add any new changes to the top(right below this line).

 - 2020-11-20
     - Role: edxapp
        - Updated the worker newrelic config to have the service variant in the app name.  This will seperate the names

>           of the newrelic apps to be `...-lms` and `...-cms` to make it easier to monitor them separately.  This will

Reviewer: Nit: I imagine our recommendation for no hard line breaks in docs would apply to markdown as well. Also understood if you want consistency within a doc.

>           impact an newrelic monitoring and alerting you have that is linked to the old app name.  It should be
>           duplicated to both of the new application names.

Reviewer: ```suggestion
          impact any newrelic monitoring and alerting you have that is linked to the old app name, which should be
          updated to use both of the new application names.
```
